pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -542,29 +542,34 @@ def instance_query(context, query, **options):
                 required=False)
 @click.option('-s', '--sort', is_flag=True, required=False,
               help='Sort by instance count. Otherwise sorted by class name')
-@add_options(namespace_option)
+@add_options(multiple_namespaces_option)
 @click.pass_obj
 def instance_count(context, classname, **options):
     """
     Count the instances of each class with matching class name.
 
-    Display the count of the instances of each CIM class whose class name
+    Displays the count of instances of each CIM class whose class name
     matches the specified wildcard expression (CLASSNAME-GLOB) in all CIM
-    namespaces of the WBEM server, or in the specified namespace
-    (--namespace option).
+    namespaces of the WBEM server, or in the specified namespaces
+    (--namespace option).  This differs from instance enumerate, etc. in that
+    it counts the instances specifically for the classname of each instance
+    returned, not including subclasses.
+
+    The CLASSNAME-GLOB argument is a wildcard expression that is matched on the
+    class names case insensitively. The special characters known from file name
+    wildcarding are supported: `*` to match zero or more characters, and `?` to
+    match a single character. To avoid shell expansion of wildcards, the
```

Comment:
It also supports "[]" ranges. In the docs description, that detail was now moved to the pattern term description, but in the help text I think we should mention it.

Reply:
DONE